engine/src/mblandroidscroller.cpp

```diff
@@ -174,11 +174,12 @@ void MCAndroidScrollerControl::SetContentRect(MCExecContext& ctxt, integer_t p_r
     // SN-2014-11-27: [[ Bug 14046 ]] m_content_rect stores user-pixel values
     m_content_rect . x = p_rect[0];
     m_content_rect . y = p_rect[1];
-    m_content_rect . width = p_rect[2];
-    m_content_rect . height = p_rect[3];
+    m_content_rect . width = p_rect[2] - p_rect[0];
+    m_content_rect . height = p_rect[3] - p_rect[1];
     
+	// PM-2016-01-14: [[Bug 16705]] Pass the correct width and height
     if (t_view != nil)
-        MCAndroidObjectRemoteCall(t_view, "setContentSize", "vii", nil, (integer_t)(t_rect.size.width - t_rect.origin.x), (integer_t)(t_rect.size.height - t_rect.origin.y));
```

Comment:
Since it is not necessary to create `t_rect` unless `t_view != nil`, maybe this would be a good opportunity to move the call to `MCNativeControlUserRectToDeviceRect()` into this `if` statement. What do you think?